bdb/bdb_osqlcur.c

```diff
@@ -193,6 +193,7 @@ int bdb_osql_shadow_is_bkfilled(bdb_cursor_ifn_t *pcur_ifn, int *bdberr)
         if (shadow->tables[cur->dbnum].ix_shadows)
             return shadow->tables[cur->dbnum].ix_shadows[cur->idx].bkfill;
         break;
+    default: break;
```

Comment:
I'd assert(0) this and similar cases below as such cases should not go unnoticed.

Reply:
Ok, changing.